sip-hdnodeid.md

```diff
@@ -0,0 +1,102 @@
+```
+SIP: hdnodeid
```

Comment:
@braydonf okay, well so far we've got `0001`, `0002`, `0003`, so obviously this one gets to be `0032`. :stuck_out_tongue_closed_eyes: